pkg/api/addons_test.go

```diff
@@ -1564,7 +1564,7 @@ func TestSetAddonsConfig(t *testing.T) {
 					Enabled: to.BoolPtr(true),
 					Config: map[string]string{
 						"omsAgentVersion":       "1.10.0.1",
-						"dockerProviderVersion": "8.0.0-3",
+						"dockerProviderVersion": "10.0.0-1",
```

Comment:
This changed in the unit test, but we didn't change the actual addon default, so the UT is failing.

Reply:
good catch.. fixed it